### PR TITLE
Responsive web design

### PR DESCRIPTION
**Pull request by dwantula**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -10,34 +10,14 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
-    <!--
```
Comment (mi-chal) [inline public/index.html:13]: Nice decision to removing this comments! 🚀

File: src/App.scss [hunk 2]
```diff
@@ -7,4 +7,45 @@ body {
...
+  .button-find-beers {
+    align-self: flex-start;
+    margin-top: 20px;
+  }
+  .random-beer {
+    margin-left: 20px;
+  }
+  .cart {
+    margin-left: -100px;
+  }
```
Comment (mi-chal) [inline src/App.scss]: If this is specific to app part it should be implemented in appropriate `.scss` file.

File: src/App.scss [hunk 3]
```diff
@@ -7,4 +7,45 @@ body {
...
+    margin-right: auto;
+  }
+  .button-find-beers {
+    align-self: flex-start;
+    margin-top: 20px;
+  }
+  .random-beer {
+    margin-left: 20px;
+  }
+  .cart {
```
Comment (mi-chal) [inline src/App.scss]: Is this typo (card)?

File: src/screens/SearchBeers/styles.scss [hunk 4]
```diff
@@ -11,10 +11,10 @@
...
-  margin-left: 860px;
-  margin-top: -145px;
-  margin-bottom: 40px;
+  margin-top: -150px;
   @include button-circle;
+  justify-content: center;
+  align-self: flex-end;
 }
 
 .button-find-beers:hover {
```
Comment (mi-chal) [inline src/screens/SearchBeers/styles.scss]: Use `&` for hover

File: src/screens/WorldBeers/WorldBeers.jsx [hunk 5]
```diff
@@ -15,7 +15,7 @@ const WorldBeersComponent = () => {
     setImageIndex(imageIndex >= images.length - 1 ? 0 : imageIndex + 1);
 
   return (
-    <div>
+    <div className="flex">
```
Comment (mi-chal) [inline src/screens/WorldBeers/WorldBeers.jsx]: Use class name specific for element.

File: src/shared/components/DeleteButton/DeleteButton.jsx [hunk 6]
```diff
@@ -6,7 +6,13 @@ import './styles.scss';
 
 const DeleteButton = ({ onClick }) => {
   return (
-    <FontAwesomeIcon icon={faTrash} className="delete-beer" onClick={onClick} />
+    <div className="cart">
```
Comment (mi-chal) [inline src/shared/components/DeleteButton/DeleteButton.jsx]: Typo?

File: src/shared/components/DeleteButton/styles.scss [hunk 7]
```diff
@@ -1,13 +1,18 @@
 @import '../../main-styles.scss';
 
+.cart {
```
Comment (mi-chal) [inline src/shared/components/DeleteButton/styles.scss]: Typo?

File: src/shared/components/FavouriteButton/FavouriteButton.jsx [hunk 8]
```diff
@@ -26,11 +26,13 @@ const FavouriteButton = ({ beerId }) => {
   }
 
   return (
-    <FontAwesomeIcon
-      icon={faThumbsUp}
-      className={active ? 'spinner-anim icon' : 'icon'}
-      onClick={addBeerIdToFavourites}
-    />
+    <div className="cart">
```
Comment (mi-chal) [inline src/shared/components/FavouriteButton/FavouriteButton.jsx]: Typo?

File: public/index.html (hunk 1 above)
Comment (dwantula) [reply public/index.html:13]: 👍

File: src/App.scss (hunk 3 above)
Comment (dwantula) [reply src/App.scss]: Yes but it's not good name this class. I change to 'icon-position'

File: src/screens/SearchBeers/styles.scss (hunk 4 above)
Comment (dwantula) [reply src/screens/SearchBeers/styles.scss]: Done

File: src/shared/components/DeleteButton/styles.scss (hunk 7 above)
Comment (dwantula) [reply src/shared/components/DeleteButton/styles.scss]: Fixed

File: src/shared/components/DeleteButton/DeleteButton.jsx (hunk 6 above)
Comment (dwantula) [reply src/shared/components/DeleteButton/DeleteButton.jsx]: Fixed!

File: src/screens/WorldBeers/WorldBeers.jsx (hunk 5 above)
Comment (dwantula) [reply src/screens/WorldBeers/WorldBeers.jsx]: Fixed

File: src/shared/components/FavouriteButton/FavouriteButton.jsx (hunk 8 above)
Comment (dwantula) [reply src/shared/components/FavouriteButton/FavouriteButton.jsx]: Yes, fiexed

File: src/App.scss (hunk 2 above)
Comment (dwantula) [reply src/App.scss]: Done!

**Pull request merged**